panel/src/components/Forms/Previews/FlagFieldPreview.vue

```diff
@@ -1,20 +1,21 @@
 <template>
-	<k-button v-bind="status" class="k-flag-field-preview" />
+	<div class="k-flag-field-preview">
+		<k-button v-if="value" v-bind="status" size="md" />
```

Comment:
Should the wrapping div remain if no value is set?

Reply:
Much better to also remove the div.